sympy/physics/control/lti.py

```diff
@@ -2296,6 +2302,11 @@ def __new__(cls, sys1, sys2=None, sign=-1):
         if not sys2:
             sys2 = TransferFunction(1, 1, sys1.var)
 
+        if isinstance(sys1, Feedback):
+            sys1 = sys1.doit()
+        if isinstance(sys2, Feedback):
+            sys2 = sys2.doit()
```

Comment:
The current design does not ever call doit() internally. The user is left to do that as needed. If the numerator and denominator could be Series or Parallel objects, then we doit() would not be needed internally.

Reply:
Yes I think it is not needed here , all tests are passing without it.